src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -419,6 +419,8 @@ def _getMethodArgs(self, args):
                 kwargs[arg] = self.get_argument(arg)
             elif isinstance(defaults[arg], list):
                 kwargs[arg] = self.get_arguments(arg) or defaults[arg]
+            elif isinstance(defaults[arg], six.integer_types):
```

Comment:
There is no longer any need to use `six` here:
```suggestion
            elif isinstance(defaults[arg], int):
```